CHANGELOG.md

```diff
@@ -148,6 +148,7 @@
 
 ## Bug Fixes
 
+- PR #5793 Fix leak in mutable_table_device_view (_descendant_storage)
```

Comment:
The changes in this PR are to `table_device_view`, not `mutable_table_device_view`. Which is it supposed to be?

Reply:
So the leak was with `mutable_table_device_view`s specifically, but the fix was to commonize the `delete`. I rephrased the changelog message here: https://github.com/rapidsai/cudf/pull/5793/commits/36c4bb4f873b715eb570ef45770b614adc9db995. 

Let me know if that's better.

Reply:
Sorry, I misunderstood that this was an indirect leak fix.